msteams-platform/tabs/tabs-link-unfurling.md

```diff
@@ -8,9 +8,9 @@ ms.localizationpriority: high
 ms.date: 06/05/2023
 ---
 
-# Open content in Stageview
+# Open content in multi-window
```

Comment:
Hi @aaronmah, We are retaining 'Stageview' as 'multi-window' would be confusing. Further in the document, we have Teams multi-window experience and it would lead to confusion if we are naming the entire experience as multi-window. The document bifurcates Stageview having three experience, so let's retain Stageview for better clarity.